pontoon/tour/static/js/tour.js

```diff
@@ -56,7 +56,17 @@ $(function () {
       function() {
         return true;
       }
-    ]
+    ],
+    listeners: {
+        beforeWizardStarts: function(){
+            $("body").addClass("tour");
+            $("body *").attr("tabindex", "-1");
```

Comment:
This means that absolutely no elements on the page can be navigated using tab. That is a big breaker for accessibility. We want to make sure that while the tour is active, no hidden element (behind the gray background) can be focused, but we want the controls of the tour to still be accessible, as well as the non-hidden elements.